README.md

```diff
@@ -925,6 +925,59 @@ Back endpoint responsible for fetching and sending information to the database r
 `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
 
 
+### Jobs
+Back endpoint responsible for fetching and sending information to the database regarding Jobs and timesheet used to clock shifts.
+
+##### GET 
+
+`api/jobs/getJobs` Gets the users current active jobs and returns them in a list.  
+
+`api/jobs/getSavedShifts/` Gets the users active shifts that have yet to be approved.
```

Comment:
"get" should not be in the endpoint name; it's already a GET.
Also, does "saved" mean "not approved" or "not submitted"?  It's an important difference.

Add "student" to name, to distinguish from staff.  It's important to be consistent with all of these.  Putting "Student" and "Staff" at end makes sense, particularly so the pairs end up next to each other in alphabetical order.